README.md

```diff
@@ -1,26 +1,32 @@
 # README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
 * Ruby version
    - 3.0.0p0 
 
-* System dependencies
-   - 
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+About the project: 
```

Comment:
you can use markdown to style the readme
https://www.markdownguide.org/basic-syntax